pkg/csi/service/vanilla/controller.go

```diff
@@ -760,6 +757,25 @@ func (c *controller) ControllerExpandVolume(ctx context.Context, req *csi.Contro
 	log := logger.GetLogger(ctx)
 	log.Infof("ControllerExpandVolume: called with args %+v", *req)
 
+	var volumePath string
+	if strings.Contains(req.VolumeId, ".vmdk") {
+		volumePath = req.VolumeId
+	}
+	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration) {
+		// Migration feature switch is enabled
+		if volumePath != "" {
+			msg := fmt.Sprintf("Cannot expand in-tree volume. :%q", volumePath)
```

Comment:
Should this be called "migrated volume"?

Reply:
changed to 

```
msg := fmt.Sprintf("Cannot expand migrated vSphere volume. :%q", volumePath)
```